src/dashboard/trade/placeOrder/OrderTypeButtons.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { ReactElement } from "react";
+import { inject, observer } from "mobx-react";
+import { ButtonGroup, makeStyles } from "@material-ui/core";
+import { OrderType } from "../../../enums";
+import { TradeStore, TRADE_STORE } from "../../../stores/tradeStore";
+import Button from "../../../common/components/input/buttons/Button";
+
+const useStyles = makeStyles(() => ({
```

Comment:
The color of those buttons should only depend on whether buy or sell is selected. If buy, both are green; if sell, both are red.